client/app/scripts/superdesk-highlights/highlights.js

```diff
@@ -79,6 +79,21 @@
                 highlight: highlight._id
             };
 
+            if (highlight.groups && highlight.groups.length > 0) {
+                var groups = [{
```

Comment:
This part looks to be a copy of the packagesService, any reason why we can't reuse that?